scripts/build-wasm.sh

```diff
@@ -15,8 +16,11 @@ ASM=wasm-crypto-asmjs/build/cjs/data.js
 # build new via wasm-pack
 echo "*** Building Rust sources"
 cd wasm-crypto
-rustup run $RUST_VER xargo build --target wasm32-unknown-unknown --release
-# rustup run $RUST_VER cargo build --target wasm32-unknown-unknown --release -Z build-std=std,panic_abort
+if [ "$RUST_VER" == "stable" ]; then
+  RUSTC_BOOTSTRAP=1 cargo build --target wasm32-unknown-unknown --release -Z build-std=std,panic_abort
```

Comment:
There is, sadly, an increase in size -

- `cargo` https://github.com/polkadot-js/wasm/runs/2678800150?check_suite_focus=true#step:3:305 `Compressed WASM: 349,002 -> 172,873 (49.53%)`
- `xargo` https://github.com/polkadot-js/wasm/runs/2678873851?check_suite_focus=true#step:3:384 `Compressed WASM: 299,511 -> 152,679 (50.98%)`